tiingo/api.py

```diff
@@ -85,59 +86,55 @@ def __init__(self, *args, **kwargs):
         }
 
         self._frequency_pattern = re.compile('^[0-9]+(min|hour)$', re.IGNORECASE)
+        self._ticker_list = None
 
     def __repr__(self):
         return '<TiingoClient(url="{}")>'.format(self._base_url)
 
-    def _is_eod_frequency(self,frequency):
+    @staticmethod
+    def _is_eod_frequency(frequency):
         return frequency.lower() in ['daily', 'weekly', 'monthly', 'annually']
 
-    # TICKER PRICE ENDPOINTS
-    # https://api.tiingo.com/docs/tiingo/daily
-    def list_tickers(self, assetTypes=[]):
-        """Return a list of dicts of metadata tickers for all supported tickers
-            of the specified asset type, as well as metadata about each ticker.
-            This includes supported date range, the exchange the ticker is traded
-            on, and the currency the stock is traded on.
-           Tickers for unrelated products are omitted.
-           https://apimedia.tiingo.com/docs/tiingo/daily/supported_tickers.zip
-           """
+    @staticmethod
+    def _get_ticker_list():
```

Comment:
We may not gain a lot in terms of readability by having this separate function when there isn't caching, and in fact we'll pay a small performance price by initializing the list twice (both here and inside `list_tickers`, whereas before that was only created once. 

I'd suggest either replacing this list comprehension with a generator comprehension ( https://www.pythonlikeyoumeanit.com/Module2_EssentialsOfPython/Generators_and_Comprehensions.html#Creating-your-own-generator:-generator-comprehensions ), or letting this part just be part of the original `list_tickers` to avoid the duplicate work.